profiler/profile.go

```diff
@@ -219,12 +219,16 @@ type limitedTraceCollector struct {
 	w       io.Writer
 	limit   int
 	written int
```

Comment:
NIT: Make this an int64? (overflow paranoia, feel free to ignore)

Reply:
Works for me :) Done.